### DISCUSSION TITLE
VR Mattermost statue garden

### DISCUSSION THREAD

**Discussion by chenilim** [mattermost/community#2] [Fun Ideas]
A VR space that shows each Mattermost user as a statue, using their VR avatar. The statues have speech bubbles with their last post in (public channel) Mattermost. You can wander the statue garden to take it all in.

Maybe instead of a statue garden, it's like a soccer (or football) stadium.

**Comment by larkox** [comment]
It would be cool. We could also keep a room in VR space for ex-employees, and another room for Community Superstars :smile:

**Comment by chetanyakan** [comment]
Kind of like a hall of fame
